packages/dialog/package.json

```diff
@@ -11,8 +11,8 @@
     "dialog"
   ],
   "dependencies": {
-    "@material/dialog": "0.43.0",
-    "@material/dom": "^0.41.0",
+    "@material/dialog": "1.0.1",
```

Comment:
can you update these to 1.1.0. Might also need to do that in the other components.